### DISCUSSION TITLE
Teleport run in K8S type NodePort ERROR: unrecognized subsystem: sftp

### DISCUSSION THREAD

**Discussion by quinhn** [gravitational/teleport#30018] [Q&A] [answered]
Hi all,
I am running Teleport v13.2.5 in K8S type NodePort.
I can access WebUI, SSH to server by web or cli successfully.
But i can not upload file to server by web or tsh scp cli.
The cli return ERROR: unrecognized subsystem: sftp.
This is error logs in Teleport:
![image](https://github.com/gravitational/teleport/assets/77380031/78213f6e-3fec-4008-b5b4-31ceab7bce92)

Please help me to find a solution or workaround!
Thanks.

**Comment by webvictim** [comment]
You mention that your cluster is on 13.2.5, but what Teleport version is your server (`172.25.210.200:3022`) running?

**Reply by quinhn** [reply, marked as answer]
Hi @webvictim,
Sorry for late respone.
I updated my servers from Teleport v10 to v13 and it's working now.
Thank for you answer!
